internal/operators/cnv/manifest.go

```diff
@@ -140,3 +152,34 @@ metadata:
   namespace: "{{.OPERATOR_NAMESPACE}}"
 spec:
   BareMetalPlatform: true`
+
+const hpp = `apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+    spec:
+      imagePullPolicy: IfNotPresent
+      storagePools:
+        - name: "sno"
+    pvcTemplate:
+      storageClassName: local-block
+      volumeMode: Block
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 50Gi
```

Comment:
there is a basic requirement for SNO, should it be added to the validations in case CNV is enabled?

Reply:
So the local-block storage class would be an LSO storage class, and the PVs created by LSO will be the size of the disk. Kubernetes will match PVs with the PVC created from this template if the PV capacity >= request size. We don't want the request size to be larger than the capacity of the LSO PV.

Reply:
Is that a dedicated disk?

Reply:
@awels in assisted, we dont want users to fail and we base everything on the discovery data. If the disk size will be 40GB, your yaml will fail, right? And the user will not be able to identify it

Reply:
Does it means we need to have also LSO ?

Reply:
@romfreiman with the yaml as it is yes, we picked 50Gi as it seemed a reasonable minimum, but we could go with 1Mi if we fear that we might have smaller disks.
@rollandf The example assumes we have LSO installed (from what I was told, assisted installer, automatically installs LSO) since we are using LSOs discovery feature to discover disks, and then using the PVs associated with those disks to mount in the host, and create directories on. But there is no reason we cannot use non LSO PVs.

Reply:
If we know that we require a minimal then we should validate it, 1M will not be really usable for the user, lets not make shortcuts here.
We have a lot of validations that we are running before the installation and we should make sure that we validate this as well